tests/muxcable_test.py

```diff
@@ -1168,7 +1168,18 @@ def test_show_muxcable_cableinfo_incorrect_logical_port_return_value(self):
         result = runner.invoke(show.cli.commands["muxcable"].commands["cableinfo"],
                                ["Ethernet0"], obj=db)
         assert result.exit_code == 1
+        
+    @mock.patch('sonic_y_cable.y_cable.get_part_number', mock.MagicMock(return_value=(False)))
+    @mock.patch('sonic_y_cable.y_cable.get_vendor', mock.MagicMock(return_value=(False)))
+    @mock.patch('show.muxcable.platform_sfputil', mock.MagicMock(return_value=1))
+    @mock.patch('utilities_common.platform_sfputil_helper.logical_port_name_to_physical_port_list', mock.MagicMock(return_value=[0]))
```

Comment:
Will the UT cover `logical_port_name_to_physical_port_list` if it's mock patched?